generators/chipyard/src/main/scala/harness/HarnessBinders.scala

```diff
@@ -237,6 +237,13 @@ class WithSimTSIOverSerialTL extends HarnessBinder({
   }
 })
 
+//TODO: Allow setting from harness with chipId argument, hardcoding is temporary hack
+class WithChipIdPinFromHarness extends HarnessBinder({
```

Comment:
Nit: Name this `WithDriveChipIdPin`. The `FromHarness` naming for clock/reset indicates that those are driven by things which are generated as part of the TestHarness, outside of HarnessBinders.